device/accton/x86_64-accton_csp7551-r0/sonic_platform/__init__.py

```diff
@@ -0,0 +1,2 @@
+__all__ = ['chassis', 'eeprom', 'platform', 'fan', 'psu']
```

Comment:
I believe all these platform files are expected to be located in the platform folder. For more details, please refer to https://github.com/Azure/SONiC/blob/master/doc/platform_api/new_platform_api.md#implementation

Reply:
Ok, I'll move these files.